apps/wallet-mobile/src/auth/CreatePinInput/CreatePinInput.tsx

```diff
@@ -74,15 +74,15 @@ const useStrings = () => {
 
 const messages = defineMessages({
   pinInputTitle: {
-    id: 'components.firstrun.custompinscreen.pinInputTitle',
+    id: 'components.initialization.custompinscreen.pinInputTitle',
```

Comment:
@MichalSzorad can you check if using crowdin is possible to rename a key? Cuz if we do it this way it will consider a new translation, I also would say that we should remove the `components` and make the feature the first level.